spec/rspec/matchers/built_in/include_spec.rb

```diff
@@ -131,6 +188,18 @@ def hash.send; :sent; end
       it_behaves_like "a Hash target"
     end
 
+    context "for a target that inherits from Hash" do
```

Comment:
This doc string doesn't describe this situation as accurately as it could.  How about:

``` ruby
context "for a target that subclasses Hash to treat string/symbol keys interchangeably, but returns a raw hash from #to_hash"
```

Pretty long, but describes the exact situation we're dealing with much more accurately.

Reply:
Much better. I change description as you proposed. Thank you.